pkgs/top-level/all-packages.nix

```diff
@@ -24413,4 +24413,5 @@ in
 
   kube3d =  callPackage ../applications/networking/cluster/kube3d {};
 
+  xygrib = callPackage ../applications/misc/xygrib/default.nix {};
```

Comment:
Please don't just append it to the bottom. Yes the file is very long and messy but we try to have some structure. Check out the first few lines of this file ;) (Your category is `### APPLICATIONS`)